src/Fileloader/extension_checker.py

```diff
@@ -0,0 +1,33 @@
+"""Module checking if input file is correct extension compared to files actual extension"""
+import os
+import magic
+
+class ExtChecker():
+    """Checking if file got correct extension type and changes it if necessary"""
+    def __init__(self, inputfile:str = None) -> None:
+        self.inputfile = inputfile
+        self.magic = magic.Magic(mime=True)
+
+    def check_file(self) -> str:
+        """Checking file extension and calling convert if not actual extension"""
+        if self.inputfile is None:
+            raise ValueError("No file has been selected")
+
+        if os.path.exists(self.inputfile) is False:
+            raise FileNotFoundError("File doesn't exist in folder")
+
+        actual_ext_type = self.magic.from_file(self.inputfile).split("/")[-1]
```

Comment:
Overvej for at smide self.magic.from_file ind i en funktion som bare læser extension typen ud og returner den